<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZSendCoinsDialog_popup</class>
 <widget class="QDialog" name="ZSendCoinsDialog_popup">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Off-line transaction signing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="frameCoinControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutCoinControl2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutCoinControl" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="lbInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Unsigned transaction:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutCoinControl1">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QTextEdit" name="teInput">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameResult">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutResult1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutResult2" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="lbResult">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Signed transaction input:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="teResult">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Steps to sign the transaction: Copy the complete transaction text in the text box above. Transfer the data to the computer that is running the off-line wallet, using a USB memory stick or other suitable media.  Select the 'Sign' menu option in the off-line wallet. In the sign dialog, paste the data into the text edit box marked: 'Unsigned transaction input'. Press the 'Sign transaction' button. If the signature was successfull, the raw hex transaction data will be displayed in the text box, starting with 'sendrawtransaction'. Copy that data, transfer it back to this on-line wallet and paste it into this text box. Press the 'Send transaction' button to submit the transaction to the PirateNetwork</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sendTransactionButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Confirm the send action</string>
       </property>
       <property name="text">
        <string>S&amp;end transaction</string>
       </property>
       <property name="icon">
        <iconset resource="../komodo.qrc">
         <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear all fields of the form.</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../komodo.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../komodo.qrc"/>
 </resources>
 <connections/>
</ui>
